docs/topics/native/native-debugging.md

```diff
@@ -174,8 +200,7 @@ Process 4985 launched: './program.kexe' (x86_64)
 ```
 
 Getting representation of the object variable (var) could also be done using the
-built-in runtime function `Konan_DebugPrint` (this approach also works for gdb,
-using a module of command syntax):
+built-in runtime function `Konan_DebugPrint` (this approach also works for GDB, using a module of command syntax):
```

Comment:
I'm not sure I understand what a "module of command syntax" means here. But this could be a gap in my knowledge about debugging?